contracts/utils/Funder.sol

```diff
@@ -0,0 +1,95 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.17;
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+
+// I wanted to build this in a way that you can also use it to fund 100 sponsor
+// wallets that each power a single self-funded Beacon.
+// Use FunderFactory to deploy.
+contract Funder is Ownable {
```

Comment:
I'm guessing it will be switched from `Ownable` to `OwnableCallForwarder` as well, right?

Reply:
SelfMulticall is for when one needs to fund multiple sponsor wallets with a single transaction. I don't see why OwnableCallForwarder would be needed.

Reply:
I guess I got confused about who the owner of this contract is going to be. I thought It would be the DAO or some multisig that might not interact with this contract directly.

Reply:
> I thought It would be the DAO or some multisig that might not interact with this contract directly.

That's correct. I still don't get what call you would have this contract forward though.

Reply:
Done in https://github.com/api3dao/airnode-protocol-v1/pull/129/commits/bd9d03f813c120f78097c5a228729825deb4c800